meilisearch-lib/src/index/search.rs

```diff
@@ -1201,7 +1328,7 @@ mod test {
         )
         .unwrap();
 
-        assert_eq!(value["title"], "Potter");
+        assert_eq!(value["title"], "Harry Potter and the Half-Blood Prince");
```

Comment:
crop 0 behavior changes, we are now considering that crop 0 is equivalent to no crop.

https://github.com/meilisearch/specifications/blob/26937397f3d943717da72a67443080d185f78b49/text/0118-search-api.md?plain=1#L265